docs/legacy/v2-products/v2-v3.md

```diff
@@ -46,7 +46,7 @@ Creating Products and Variants on V3:
 
 Variants can be included with a GET request to lower the number of API calls being made using `?include=variants`.
 
-V3 includes an endpoint for creating a [Catalog Tree](/api-reference/catalog/catalog-api).
+V3 includes endpoints for working with catalog trees. Stores that have multi-storefront enabled can have more than one tree. See the catagories section of the [Multi-Storefront API Guide](/api-docs/multi-storefront/api-guide#categories). Stores that do not have multi-storefront enabled ...
```

Comment:
@markcmurphy Does the Catagory tree page [here](https://developer.bigcommerce.com/docs/ZG9jOjQ4MjA4MjI2-multi-storefront-api-guide#categories) apply to both non-MSF and MSF?  

On the MSF guide, it mentions that "Previously, a store had a collection of categories that were organized in a tree structure. This collection of categories has been abstracted from the store and converted into the store's first category tree." Do both MSF and non-MSF use the same catagory tree endpoints?

Reply:
@bc-andreadao IIRC we removed the other category tree endpoints because they didn't work. @markcmurphy is this right?

Reply:
@bc-andreadao If a store only has a single storefront and isn't using MSF, then developers don't need to concern themselves with category trees. Non-MSF accounts can use that same endpoint, though it is just going to return 
```  json 
        {
            "id": 1,
            "name": "Default catalog tree",
            "channels": [
                1
            ]
        }
```